src/aurora/xmlfixer.h

```diff
@@ -44,13 +44,14 @@ class XMLFixer {
 	Common::SeekableReadStream *fixXMLStream(Common::SeekableReadStream &in);
 	
 private:
-	typedef std::vector<Common::UString>	ElementList;
+	typedef std::vector<Common::UString>	ElementList;	// List of elements
+	typedef std::vector<Common::UString>	SegmentList;	// List of element segments

```

Comment:
There should be a space between `std::vector<Common::UString>` and `SegmentList` (and `ElementList`), not a tab.